### DISCUSSION TITLE
Direct @mention profile user

### DISCUSSION THREAD

**Discussion by rei-zero246** [robertodoering/harpy#838] [Q&A] [answered]
Is there a way that when you are on someone's TL that you can @ mention them or compose a fresh tweet to that person or entity?

**Comment by rei-zero246** [comment]
I think I understand why this is the case.

**Comment by robertodoering** [comment, marked as answer]
There is no way to compose a new Tweet with a pre-added @mention from a user profile yet, but this is something that should be added in the future
